data-roadmap/data_roadmap/utils/utils_data_set_description_schema.py

```diff
@@ -0,0 +1,154 @@
+import pathlib
+
+import yamale
+import yaml
+
+import importlib_resources
+
+DATA_ROADMAP_DIRECTORY = importlib_resources.files("data_roadmap")
+UTILS_DIRECTORY = DATA_ROADMAP_DIRECTORY.joinpath("utils")
+DATA_SET_DESCRIPTIONS_DIRECTORY = DATA_ROADMAP_DIRECTORY.joinpath(
+    "data_set_descriptions"
+)
+
+DATA_SET_DESCRIPTION_SCHEMA_FILE_PATH = DATA_ROADMAP_DIRECTORY.joinpath(
+    "data_set_description_schema.yaml"
+)
+DATA_SET_DESCRIPTION_FIELD_DESCRIPTIONS_FILE_PATH = DATA_ROADMAP_DIRECTORY.joinpath(
+    "data_set_description_field_descriptions.yaml"
+)
+
+DATA_SET_DESCRIPTION_TEMPLATE_FILE_PATH = DATA_ROADMAP_DIRECTORY.joinpath(
+    "data_set_description_template.yaml"
+)
```

Comment:
small nit but you can do 
```py
UTILS_DIRECTORY  = DATA_ROADMAP_DIRECTORY / "utils"
```
for this and all the other `joinpath`s -- with `pathlib` its equivalent!

Reply:
Ooooooh nice! Sexy.